src/filter_tree/filter_tree.h

```diff
@@ -96,6 +96,12 @@ rax *FilterTree_CollectAttributes(const FT_FilterNode *root);
 /* Checks to see if tree contains given operation. */
 bool FilterTree_containsOp(const FT_FilterNode *root, AST_Operator op);
 
+/* Checks to see if tree contains given function.
+ * root - tree root to traverse.
+ * func - function name to lookup.
+ * node - point to tree node in which func was located, null if func isn't located. */
```

Comment:
The `node` argument populated in this function is not used by any of its callers at present. It seems like it could be handy, so I don't have an opinion on whether it should be removed or not, but I wanted to point out the option!

Reply:
yes I know.
if it ain't broken...

Reply:
:+1: That's fine by me!